src/Validators/ExpenseDtoValidator.cs

```diff
@@ -0,0 +1,29 @@
+using FluentValidation;
+using server.Dtos;
+
+namespace server.Validators;
+
+public class ExpenseDtoValidator : AbstractValidator<ExpenseDto>
+{
+    public ExpenseDtoValidator()
+    {
+        RuleFor(expense => expense.Amount)
+            .NotEmpty();
+        RuleFor(expense => expense.Date)
+            .NotEmpty()
+            .Must(BeAValidDate)
+            .WithMessage("Please provide a valid date");
+        RuleFor(expense => expense.Note)
+            .MaximumLength(100)
```

Comment:
why maximum length is 100 ?

Reply:
This is a temporary solution, we don’t know yet what max length should be